src/annotationLayer.js

```diff
@@ -172,16 +183,118 @@ var annotationLayer = function (args) {
     }
   };
 
+  /**
+   * Select or deselect an edit handle.
+   *
+   * @param {geo.event} evt The mouse move event.
+   * @param {boolean} enable Truthy to select the handle, falsy to deselect it.
+   * @returns {this}
+   */
+  this._selectEditHandle = function (evt, enable) {
+    if (!evt.data || !evt.data.editHandle) {
+      return;
+    }
+    $.each(m_features[geo_annotation._editHandleFeatureLevel], function (type, feature) {
+      feature.feature.modified();
+    });
+    m_this.currentAnnotation.selectEditHandle(evt.data, enable);
```

Comment:
curious...here we are passing enable as it is and 2 lines below we are converting into true boolean representation. I think I may know why but just double checking.

Reply:
Most of javascript handles truthy or falsy values they way you'd expect.  However, jQuery's toggleClass does *not*, and must take either an actual boolean or a function that returns a boolean.

Reply:
ah, thanks good to know.